terraform/ec2/linux/variables.tf

```diff
@@ -93,15 +88,6 @@ variable "cwa_github_sha_date" {
   type    = string
   default = ""
 }
-variable "performance_number_of_logs" {
```

Comment:
This may need a change in the integ test yaml since terraform throws errors when you send a var not in the variables.

Reply:
Yes that's fine. EC2Linux does not use it. Only [Performance](https://github.com/aws/amazon-cloudwatch-agent/blob/main/.github/workflows/integrationTest.yml#L557-L574) use it